docker/mockserv.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:1.20-alpine AS build-env
+FROM --platform=$BUILDPLATFORM golang:1.21-alpine AS build-env
```

Comment:
should be merged if we bump in https://github.com/rollkit/rollkit/blob/main/go.mod#L3

Reply:
bumped in main, we can merge